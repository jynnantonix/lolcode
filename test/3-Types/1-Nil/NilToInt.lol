This test makes sure a nil typed value can NOT be cast to an integer typed value.

HAI
	I HAS A var, BTW var is NOOB
	SUM OF 0 AN var
KTHXBAI
