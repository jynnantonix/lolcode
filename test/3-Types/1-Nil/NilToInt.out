Unexpected error: unsupported operand type(s) for +: 'int' and 'NoneType'
